### PR TITLE
Update tancik2020fourier_1.md

### PR DESCRIPTION
**Pull request by ayushchauhan**

### CODE REVIEW

File: _summaries/tancik2020fourier_1.md [hunk 1]
```diff
@@ -1,19 +1,91 @@
...
+
+
+### Application of Fourier feature mapping
+
+In real-world problems of higher dimensions, it is not possible to densely sample the Fourier basis frequencies for the feature mapping because of scalability issues. Therefore, the paper proposes to sample a smaller set of random Fourier features from a parametric distribution which matches the performance of the dense set of Fourier features. It further finds that the performance does not depend so much on the family of the distribution (distribution shape) but on its scale (standard deviation). Proved empirically in the figure below. The three subfigures are experiments with three different synthetic datasets.
+
```
Comment (biofizzatreya) [inline _summaries/tancik2020fourier_1.md:70]: You can maybe add a line here to say, among the ones they tested Gaussian provided better results among a wider range of standard deviations.

File: _summaries/tancik2020fourier_1.md [hunk 2]
```diff
@@ -1,19 +1,91 @@
...
+author: ayushchauhan
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## The core idea
```
Comment (biofizzatreya) [inline _summaries/tancik2020fourier_1.md:10]: Very nice summary overall. Really enjoyed reading it.

File: _summaries/tancik2020fourier_1.md [hunk 3]
```diff
@@ -1,19 +1,91 @@
...
+
+The paper proposes that using a Fourier feature mapping of the inputs allows a coordinate-based MLP to learn **high frequency functions** in low dimension problems such as images and 3D objects. The Fourier features are a more general case of the sinusoidal position encoding. It shows using kernel regression (with a Neural Tangent Kernel (NTK)) that standard MLPs without Fourier features are not well suited for coordinate-based tasks such as shape representation, texture synthesis, shape inference, novel view synthesis because of their **spectral bias** towards low frequency functions.
+
+Leveraging literature that approximates deep networks with kernel regression, the paper theoretically explains the reason for this spectral bias and then shows how the Fourier features help in overcoming the problem. Specifically, it proves that the Fourier features transform the NTK into a stationary kernel with tunable frequency components -- thus allowing the network to learn high frequency functions. It also proves the above claims empirically through a few simple experiments.
+
```
Comment (biofizzatreya) [inline _summaries/tancik2020fourier_1.md:15]: The summary could be a bit long, especially if you explain the equations. So I would suggest shortening the intro a bit. e.g. this line "coordinate-based tasks such as shape representation, texture synthesis, shape inference, novel view synthesis" you can only give one example.

**Review by biofizzatreya — COMMENTED** [review]
Review

File: _summaries/tancik2020fourier_1.md (hunk 1 above)
Comment (ayushchauhan) [reply _summaries/tancik2020fourier_1.md:70]: You mean "Gaussian provided better results among a wider range of distributions" or "Gaussian provided better results over a wider range of standard deviation"? And where does the paper say any of the two?

File: _summaries/tancik2020fourier_1.md (hunk 3 above)
Comment (ayushchauhan) [reply _summaries/tancik2020fourier_1.md:15]: Done.

**Pull request merged**